test/Passes/pack-folding.mlir

```diff
@@ -0,0 +1,33 @@
+// RUN: tpp-opt %s -constant-fold-pack -canonicalize | FileCheck %s
```

Comment:
would `-cse` be enough?

Reply:
Instead of canonicalize? Doesn't matter much here, to be honest...